core/template.go

```diff
@@ -23,6 +24,17 @@ var TemplateFuncsNoColor = map[string]interface{}{
 	},
 }
 
+// envColorDisabled returns if output colors are forbid by environment variables
+func envColorDisabled() bool {
+	return os.Getenv("NO_COLOR") != "" || os.Getenv("CLICOLOR") == "0"
+}
+
+// envColorForced returns if output colors are forced from environment variables
+func envColorForced() bool {
+	val, ok := os.LookupEnv("CLICOLOR_FORCE")
+	return ok && val != "0"
```

Comment:
Happy to change this to match the reference if that's preferred, but wanted to share a way to check if the environment variable is present then check for a `!= 0` value instead of checking that the variable isn't the empty value.